x/cork/keeper/keeper.go

```diff
@@ -271,59 +270,51 @@ func (k Keeper) GetCorkResults(ctx sdk.Context) []*types.CorkResult {
 // Votes //
 ///////////
 
-func (k Keeper) GetApprovedScheduledCorks(ctx sdk.Context, currentBlockHeight uint64, threshold sdk.Dec) (approvedCorks []types.Cork) {
-	corksForBlockHeight := make(map[uint64][]types.Cork)
-	corkPowersForBlockHeight := make(map[uint64][]uint64)
-
+func (k Keeper) GetApprovedScheduledCorks(ctx sdk.Context) (approvedCorks []types.Cork) {
```

Comment:
I am guessing this change is because the function was always used with current height?

Reply:
Yes